pkg/controller/node/node_controller.go

```diff
@@ -1233,14 +1234,21 @@ func (ctrl *Controller) updateCandidateNode(mosc *mcfgv1.MachineOSConfig, mosb *
 		}
 
 		lns := ctrlcommon.NewLayeredNodeState(oldNode)
+		desiredConfig := upgrademonitor.NotYetSet
```

Comment:
```suggestion
		var desiredConfig string
```

This assign won't do anything, and I suspect our verify will complain